release-notes/unreleased/1931.deprecation.rst

```diff
@@ -0,0 +1,6 @@
+The ``channel_strategy`` parameter has been deprecated.
+The Q-CTRL Performance Management strategy will be removed on October 18th. 
+To continue using Q-CTRL in your workflow, please explore the following options:
+
+- Qiskit Functions Catalog: https://quantum.ibm.com/functions
+- Fire Opal: https://q-ctrl.com/fire-opal
```

Comment:
We can put more words in the release notes:

 - If your organization has an existing IBM Quantum Premium Plan instance: migrate to the Q-CTRL Performance Management Function, found in the [Qiskit Functions Catalog](http://quantum.ibm.com/functions).

- To continue using Qiskit Runtime with IBM Cloud: migrate to Q-CTRL [Fire Opal](https://q-ctrl.com/fire-opal), the same performance management product accessible directly through Q-CTRL. You can [connect your IBM Cloud API key and Qiskit Runtime CRN](https://docs.q-ctrl.com/fire-opal/discover/hardware-providers/how-to-authenticate-with-ibm-credentials) to Fire Opal.